SonarQube.TeamBuild.PreProcessor/Roslyn/NuGetAnalyzerInstaller.cs

```diff
@@ -1,203 +0,0 @@
-﻿//-----------------------------------------------------------------------
-// <copyright file="NuGetAnalyzerInstaller.cs" company="SonarSource SA and Microsoft Corporation">
-//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
-//   Licensed under the MIT License. See License.txt in the project root for license information.
-// </copyright>
-//-----------------------------------------------------------------------
-
-using NuGet;
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
-using System.Linq;
-
-namespace SonarQube.TeamBuild.PreProcessor.Roslyn
-{
-    public class NuGetAnalyzerInstaller : IAnalyzerInstaller
```

Comment:
I like your thinking :-) Unfortunately, they don't need it - they're only tweaking the project configuration, not actually fetching the packages.